data/settings/icon-grid-C.json.in

```diff
@@ -3,24 +3,41 @@
     "chromium-browser.desktop",
     "org.gnome.Nautilus.desktop",
     "libreoffice-writer.desktop",
-    "rhythmbox.desktop",
-    "com.endlessm.photos.desktop",
+    "libreoffice-calc.desktop",
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.en.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
-    "eos-folder-games.directory",
-    "eos-folder-work.directory",
     "eos-folder-curiosity.directory",
-    "eos-folder-social.directory"
+    "eos-folder-games.directory",
+    "eos-folder-social.directory",
+    "net.minetest.Minetest.desktop",
```

Comment:
This is intentional and indeed expected, whenever the user adds an app to the desktop it'll be in the 'last' slot so this scenario has to look good, assuming we're successful in our goals.